apps/src/gamelab/AnimationTab/AnimationTab.jsx

```diff
@@ -13,6 +15,13 @@ import AnimationList from './AnimationList';
 import FrameList from './FrameList';
 import ResizablePanes from './ResizablePanes';
 
+/**
+ * @const {string} domain-relative URL to Piskel index.html
+ * In special environment builds, append ?debug flag to get Piskel to load its own debug mode.
+ */
+const PISKEL_PATH = '/blockly/js/piskel/index.html' +
```

Comment:
This makes me want to go rename the `[cdo]/dashboard/public/blockly` symlink to `[cdo]/dashboard/public/apps`.  I wonder how much stuff that would break.